### PR TITLE
Add ERC: Tokenized Vaults with Lock-in Period

### PR DESCRIPTION
**Pull request by Ankarrr**
This is migrated from https://github.com/ethereum/EIPs/pull/6229.

This draft ERC is an extension of the **ERC-4626 Tokenized Vaults** designed to support the lock-in period.

I used the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

### CODE REVIEW

**Comment by Ankarrr** [comment]
> ### File `ERCS/erc-6229.md`
> Requires 1 more reviewers from @axic, @g11tech, @SamWilsn, @xinbenlv

Could someone please review my PR? Thank you very much!

**Comment by Ankarrr** [comment]
Hey @axic, @g11tech, @SamWilsn, @xinbenlv,

This EIP extends the ERC-4626 Tokenized Vaults with the lock-in period. It has been reviewed at https://github.com/ethereum/EIPs/pull/6229, and all issues have been resolved. I believe this PR can be easily reviewed. I would appreciate it if you could help me review this.

Please let me know if you have any questions. Thank you!

Sincerely,
Anderson

File: ERCS/erc-6229.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
+---
+eip: 6229
+title: Tokenized Vaults with Lock-in Period
+description: ERC-4626 Tokenized Vaults with Lock-in Period.
```
Comment (SamWilsn) [inline ERCS/erc-6229.md:4]: Your description doesn't add much new information that wasn't already in the title. Try to elaborate on the idea you introduce there.

**Review by SamWilsn — APPROVED** [review]
Fix the one comment at your leisure.

File: ERCS/erc-6229.md (hunk 1 above)
Comment (Ankarrr) [reply ERCS/erc-6229.md:4]: I see. I'll try to elaborate more information in description.

**Pull request merged**